packages/fxa-payments-server/server/config/index.js

```diff
@@ -14,7 +14,7 @@ convict.addFormats(require('convict-format-with-validator'));
 const conf = convict({
   featureFlags: {
     subscriptionCoupons: {
-      default: false,
+      default: true,
```

Comment:
Oh, maybe this should default to false since it affects end use UI.  The auth-server flag is probably ok with a default of true.